src/index.js

```diff
@@ -95,11 +95,11 @@ interface Platform {
   /**
   * The runtime's location API.
   */
-  location: Object;
+  location: typeof window.location;
   /**
   * The runtime's history API.
   */
-  history: Object;
+  history: typeof window.history;
```

Comment:
urr... can we check this. Because I was under the impression `typeof Foo` meant ctor of Foo. Because, 

```ts
class Foo {
}
let Foo2:Foo = Foo;
let foo = new Foo2();
``` 
is **not valid**, while
```ts
let Foo2: typeof Foo = Foo;
let foo = new Foo2();
```
**is valid**, and 
```ts
let foo = new Foo();
let Foo2: typeof Foo = foo;
``` 
is **not allowed**.